src/python/if-setninger/if-setninger.md

```diff
@@ -0,0 +1,106 @@
+---
+title: If-setninger
+level: 2
+author: Ole Kristian Pedersen, Kodeklubben Trondheim
+---
+
+# Introduksjon {.intro}
+Vi har allerede lært at vi kan ta bruke `input()` for å få tekst fra
+brukeren og `print()` for å skrive ut tekst til brukeren. Det kan for
+eksempel gjøres slik:
+
+```python3
+navn = input("Hva er navnet ditt? ")
+print("Hei, " + navn)
+```
+
+I denne oppgaven skal vi lære hvordan vi kan bruke if-setninger for å la
+programmet bestemme hva det skal gjøre, helt avhengig av hva brukeren
+skriver inn.
```

Comment:
I denne oppgaven skal vi lære hvodan vi bruker `if`-setninger for å bestemme hva programmet skal gjøre, ...